src/asound/bluealsa-ctl.c

```diff
@@ -137,6 +143,99 @@ static const char *soft_volume_names[] = {
 	"software",
 };
 
+static const struct {
+	const char *name;
+	int16_t value;
+} delay_adjustment_map[] = {
```

Comment:
How about making sync a slider instead of enum control element?

Disclaimer: I have not check your branch yet, so maybe indeed enum is a better choice. I'm mostly concerned about the maintenance. If enum is better, how about generating such a list dynamically?

Reply:
>How about making sync a slider instead of enum control element

AFAIK `alsamixer` always shows volume type controls as 0-100%, which is not appropriate for delay adjustment. It also colours lowest 40% of bar height, middle 40% white and top 20% red, which is also not appropriate in this case. If  there is some way to persuade it to show -37250 to +37250 (and without colouring) then I will change to slider.

> how about generating such a list dynamically?

I guess that would not be too difficult, although I haven't done any work on that.

Reply:
> AFAIK alsamixer always shows volume type controls as 0-100%, which is not appropriate for delay adjustment

OK, fair enough :)